x-pack/plugins/index_lifecycle_management/public/locator.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SerializableState } from 'src/plugins/kibana_utils/common';
+import { LocatorDefinition } from '../../../../src/plugins/share/public/';
+import {
+  getPoliciesListPath,
+  getPolicyCreatePath,
+  getPolicyEditPath,
+} from './application/services/navigation';
+import { ManagementAppLocator } from '../../../../src/plugins/management/common';
```

Comment:
nit: could you please shorten the import url similar to line 8 and move `src` imports higher than ILM app imports?

Reply:
done